zcash_client_sqlite/src/chain.rs

```diff
@@ -18,18 +25,18 @@
 //! //
 //! // Given that we assume the server always gives us correct-at-the-time blocks, any
 //! // errors are in the blocks we have previously cached or scanned.
-//! if let Err(e) = validate_combined_chain(&db_cache, &db_data) {
+//! if let Err(e) = validate_combined_chain(network, &db_cache, &db_data) {
 //!     match e.kind() {
 //!         ErrorKind::InvalidChain(upper_bound, _) => {
 //!             // a) Pick a height to rewind to.
 //!             //
 //!             // This might be informed by some external chain reorg information, or
 //!             // heuristics such as the platform, available bandwidth, size of recent
 //!             // CompactBlocks, etc.
-//!             let rewind_height = upper_bound - 10;
+//!             let rewind_height = upper_bound.sub(10);
```

Comment:
```suggestion
//!             let rewind_height = upper_bound - 10;
```
(This might require adding an `impl Sub<..> for &BlockHeight`, or modifying the `impl Sub<..> for BlockHeight` to be generic over `T: Borrow<BlockHeight>`)